src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouteError, isRouteErrorResponse } from 'react-router-dom';
+
+export default function ErrorBoundary() {
+    const error = useRouteError();
```

Comment:
Try to separate single line statements from multiline statements / code blocks to improve readability